### PR TITLE
Bash completion

### PR DESCRIPTION
**Pull request by crownedgrouse**
Add Bash completion script for Erlang.mk specific make targets.
Templates completion included.

### CODE REVIEW

**Comment by essen** [comment]
Maybe `etc` instead of `misc`.

**Comment by crownedgrouse** [comment]
OK I will fix

File: etc/bash_completion [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020, Loïc Hoguin <essen@ninenines.eu>
```
Comment (essen) [inline etc/bash_completion:2]: I'm not the author of this file. Please update the copyright.

Comment (crownedgrouse) [reply etc/bash_completion]: done.

**Comment by essen** [comment]
Merged, thanks!

**Comment by crownedgrouse** [comment]
Erlang.mk website not up-to-date. What is update frequency? Manual or automatic?

**Comment by essen** [comment]
Manual. I'll do it in a minute.

**Pull request closed without merging**